test/Serilog.Tests/Serilog.Tests.csproj

```diff
@@ -16,19 +16,15 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.10.0" />
+    <PackageReference Include="System.Reactive.Linq" Version="6.0.1" />
     
-    <PackageReference Include="xunit" Version="2.3.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.3.0" />
+    <PackageReference Include="xunit" Version="2.8.1" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.8.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
     
-    <PackageReference Include="Rx-Main" Version="2.2.5" />
```

Comment:
```Rx-Main``` seems to have been delisted from nuget.org, so I've tried changing it to System.Reactive.Linq (only effects the test project)